travis-ci/run_task_inside_docker.sh

```diff
@@ -78,24 +109,54 @@ elif [[ $TASK == *rpms ]]; then
     # for globus-gridftp-server:
     packages+=(fakeroot)
     # for globus-xio-udt-driver:
-    packages+=(udt udt-devel glib2-devel libnice-devel gettext-devel libffi-devel)
+    if [[ $OS == *9 ]]; then
+
+        # libnice-devel is not available for CentOS Stream 9 / Rocky Linux 9.
+        #
+        # make_rpms.sh was also updated in this regard.
+        :
+    else
+        packages+=(udt udt-devel glib2-devel libnice-devel gettext-devel libffi-devel)
+    fi
     # for globus-gram-job-manager:
     packages+=(libxml2-devel)
     # for myproxy:
     packages+=(pam-devel voms-devel cyrus-sasl-devel openldap-devel voms-clients initscripts)
     # for globus-net-manager:
-    packages+=(python-devel)
+    if [[ $OS == *7 ]]; then
+
+        packages+=(python-devel)
+    else
+        packages+=(python3-devel)
+    fi
     # for globus-gram-audit:
     packages+=('perl(DBI)')
     # for globus-scheduler-event-generator:
-    packages+=(redhat-lsb-core)
+    if [[ $OS == *9 ]]; then
+
+        # redhat-lsb-core is not available for CentOS Stream 9 / Rocky Linux 9.
+        # But the default is also to not use LSB, so can be ignored.
+        :
+    else
+        packages+=(redhat-lsb-core)
+    fi
     # for myproxy-oauth
-    packages+=(m2crypto mod_ssl mod_wsgi pyOpenSSL python-crypto)
+    if [[ $OS == *7 ]]; then
+
+        packages+=(m2crypto mod_ssl mod_wsgi pyOpenSSL python-crypto)
+    else
+        # Except for mod_ssl above packages are not available on CentOS Stream
```

Comment:
I guess you would need python3-pyOpenSSL, python3-cryptography, python3-m2crypto and python3-mod_wsgi
?

Reply:
I've in the mean time opened PR #200 which might make the code-base of myproxy-oauth use python3

Reply:
Commented on that in #200.